boards/arm/qemu_cortex_m3/qemu_cortex_m3.yaml

```diff
@@ -5,5 +5,7 @@ arch: arm
 toolchain:
   - zephyr
   - gccarmemb
+supported:
+  - slip
```

Comment:
Testing the network stack happens as best it happens with the given "physical" interface we utilize.  With QEMU the "physical" interface is SLIP.  We aren't testing ethernet functionality with qemu or wifi.  If we had those in qemu we'd add "- wifi", "- ethernet" to the supported: list.

Saying 'net' is meaningless at this point